pallets/common/src/benchs/asset.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{
+    benchs::User,
+    constants::currency::POLY,
+    traits::{
+        asset::{AssetFnTrait, AssetName, AssetType},
+        identity::Trait as IdentityTrait,
+    },
+};
+
+use polymesh_primitives::{ticker::TICKER_LEN, Ticker};
+
+use frame_system::RawOrigin;
+use sp_std::{convert::TryFrom, vec};
+
+/// Create a ticker and register it.
+pub fn make_ticker<Asset, Balance, Acc, O, N>(owner: O, opt_name: Option<N>) -> Ticker
```

Comment:
Another possibility would be to return DispatchResult and convert `Asset::register_ticker(owner, ticker).unwrap()` to `Asset::register_ticker(owner, ticker)?`, which would allow to have meaningful error messages in benches.

Reply:
@vkomenda  I cannot use directly the `DispathResult` type, but I've just pushed a solution with regular `Result<....>`. 
Let me know what you thing.